doc/usecases/record_elektra/UC_ansible_export.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Exporting configuration as Ansible playbook
+
+## Summary
+
+- **Title:** Exporting configuration as Ansible playbook
+- **Scope:** Configuration
+- **Level:** User goal
+- **Actors:** User (usually a sysadmin)
+- **Brief:** User exports certain configuration as Ansible playbook
+
+## Scenarios
+
+- **Precondition:** Some kind of configuration has to exist
+  
+- **Main success scenario:** 
+  - User specifies which configuration to include in playbook
+  - It must be possible to select only configuration that has changed (was recorded in session recording)
+  - Ansible playbook is generated
+  
+- **Alternative scenario:** When no configuration exists, an empty playbook is generated
+- **Error scenario:** -
+- **Postcondition:** 
+  - Ansible playbook exists
```

Comment:
This UC does not catch the situation that a playbook is updated (already existed before and gets updated with what was recorded).

Reply:
Wouldn't this be another use case? Merging of playbooks?